src/node/index.ts

```diff
@@ -14,11 +18,13 @@ import { debounce } from 'perfect-debounce'
 import colors from 'picocolors'
 import type { RollupOutput } from 'rollup'
 import type { InlineConfig } from 'vite'
+import liveReloadPlugin from 'vite-plugin-live-reload'
```

Comment:
Please use https://github.com/ElMassimo/vite-plugin-full-reload instead. It is an improved version upon the work by @arnoson.

You can import the type directly instead of inferring it.

Reply:
Will do!